templates/git-sync-relay/git-sync-relay-deployment.yaml

```diff
@@ -93,4 +93,11 @@ spec:
           env:
             - name: GIT_ROOT
               value: /git
+          livenessProbe:
+            exec:
+              command:
+              - touch
+              - /git/.git/git-daemon-export-ok
+            initialDelaySeconds: 5
+            periodSeconds: 15
```

Comment:
I know this is only a single filesystem operation, but maybe this is too frequent?

Reply:
Yeah, this isn't going to matter at all with a 1s local git sync period.